tools/run_tests/run_tests.py

```diff
@@ -893,6 +899,37 @@ def runs_per_test_type(arg_str):
         msg = '\'{}\' is not a positive integer or \'inf\''.format(arg_str)
         raise argparse.ArgumentTypeError(msg)
 
+
+def _python_config_generator(name, major, minor, bits, config_args):
+  return PythonConfig(
+    name, config_args['shell'] + config_args['builder'] + config_args['builder_prefix_arguments'] +
```

Comment:
My eyes are melting! Please wrap at eighty columns (even if there's other code in this file that doesn't).